test/Confluent.SchemaRegistry.Serdes.IntegrationTests/Tests_Json/Util.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections.Generic;
+
+namespace Confluent.SchemaRegistry.Serdes.IntegrationTests
+{
+    public class Util
+    {
+        public static List<string> GetRandomJsonSchemas(int num)
+        {
+            var schemas = new List<string>();
+            for (int i=0; i<num; ++i)
+            {
+                // var schema = ;
+            }
+            return schemas;
+        }
+    }
+}
+
+
+
+
+
+//   public static List<String> getRandomJsonSchemas(int num) {
```

Comment:
A function that does nothing and a lot of commented-out code, Im thinking this might be a leftover?